release-notes/opensearch-release-notes-2.19.0.md

```diff
@@ -0,0 +1,1163 @@
+
+* A built-in multimodal preprocessor function for Cohere lets users create a connector by providing a function name, simplifying development and avoiding complex Painless scripts.
+
+* An enhancement to the conversational agent allows users to build tools using large language model (LLM)-generated inputs as search parameters.
+
+* A new Learning to Rank plugin enhances search relevance by incorporating ML and behavioral data. It leverages lightweight ML models like XGBoost and RankLib to rescore search results, considering multiple factors in the ranking process to significantly improve the accuracy and relevance of results.
+
+* A new `template` query type introduces placeholder variables in search queries that remain unresolved until a search request processor assigns their value, enabling more dynamic and secure search operations.
+
+
+#### EXPERIMENTAL FEATURES
+
+OpenSearch 2.19 includes the following experimental functionality. Experimental features are disabled by default. For instructions on how to enable them, refer to the documentation for the feature.
+
+* The experimental Discover view now supports the SQL and Piped Processing Language (PPL) query languages in addition to Dashboards Query Language (DQL) and Lucene, along with new autocomplete functionality and improved data selection to provide a more flexible and efficient query experience.
+
+* Star-tree aggregation adds support for metric aggregations and date histograms with metric aggregations, delivering up to 100x query reduction and 30x lower cache utilization, with optimized binary search for multiple `terms` queries.
+
+* Improvements to disk-tiered request cache performance divide the cache into multiple partitions, each protected by its own read/write lock, enabling multiple concurrent readers to access data without contention and multiple writers to operate simultaneously, providing higher write throughput.
+
+* Experimental derived source functionality for k-NN vectors optimizes storage by removing vectors from the JSON source and dynamically injecting them when needed, supporting flat vector mappings, object fields, and single-level nested fields.
+
+**DEPRECATION NOTICES**
+
+**Deprecating support for Ubuntu Linux 20.04**
+Please note that OpenSearch will deprecate support for Ubuntu Linux 20.04 as a continuous integration build image and supported operating system in an upcoming version, as Ubuntu Linux 20.04 will reach end-of-life with standard support as of April 2025 (refer to [this notice](https://ubuntu.com/blog/ubuntu-20-04-lts-end-of-life-standard-support-is-coming-to-an-end-heres-how-to-prepare) from Canonical Ubuntu). For a list of OpenSearch's compatible operating systems, [visit here](https://opensearch.org/docs/latest/install-and-configure/os-comp/).
+
+**Deprecating support for features and plugins in OpenSearch 3.0.0**
```

Comment:
Can we use `####` for deprecation notices as well?
That would be easier for us to save the anchor link.

Thanks.

Reply:
Done!